faq-bundle/src/Resources/contao/modules/ModuleFaqReader.php

```diff
@@ -124,18 +127,16 @@ protected function compile()
 		$this->Template->before = false;
 
 		// Add image
-		if ($objFaq->addImage && $objFaq->singleSRC)
+		if ($objFaq->addImage && null !== ($figureBuilder = $this->getFigureBuilderIfResourceExists($objFaq->singleSRC)))
 		{
-			$objModel = FilesModel::findByUuid($objFaq->singleSRC);
-
-			if ($objModel !== null && is_file(System::getContainer()->getParameter('kernel.project_dir') . '/' . $objModel->path))
-			{
-				// Do not override the field now that we have a model registry (see #6303)
-				$arrFaq = $objFaq->row();
-				$arrFaq['singleSRC'] = $objModel->path;
-
-				$this->addImageToTemplate($this->Template, $arrFaq, null, null, $objModel);
-			}
+			$figureBuilder
+				->setSize($objFaq->size)
+				->setMetaData($objFaq->getOverwriteMetaData())
+				->enableLightbox($objFaq->fullsize)
+				->build()
+				->applyLegacyTemplateData($this->Template, $objFaq->imagemargin, $objFaq->floating);
+
+			$this->Template->useImage = true;
```

Comment:
Same here?

Reply:
fixed in d70f0466606df10fdfff2f1da509aef761c37d4c